core/contractsdk/java/example/counter/src/main/java/com/baidu/xuper/example/Counter.java

```
@@ -0,0 +1,46 @@
+package com.baidu.xuper.example;
+
+import java.math.BigInteger;
+
+import com.baidu.xuper.Context;
+import com.baidu.xuper.Contract;
+import com.baidu.xuper.ContractMethod;
+import com.baidu.xuper.Driver;
+import com.baidu.xuper.Response;
+
+/**
+ * Counter
+ */
+public class Counter implements Contract {
+
+    @Override
+    @ContractMethod
+    public Response initialize(Context ctx) {
+        return Response.ok("ok".getBytes());
+    }
+
+    @ContractMethod
+    public Response increase(Context ctx) {
+        byte[] key = ctx.args().get("key");
+        if (key == null) {
+            return Response.error("missing key");
```

Maybe throwing exception rather than returning is a better way.